CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.4.0] - 2022-10-07
+
+* Fixed
+  * Default to TLS 1.2 for all connections
```

Comment:
```suggestion
  * Default to TLS 1.2 for all connections. This fixes issues if TLS 1.3 is used or attempted. See xxx...
```
Not sure what the right URL is, but I wanted to get a link there to help find the details.